### PR TITLE
optionally use compass

### PR DESCRIPTION
**Pull request by koriroys**
I don't know if a config var is the best way, but it was the first thing I thought of.

Is there a better way to check for compass?

### CODE REVIEW

**Comment by koriroys** [comment]
@tonycoco updated per your comments. check it

**Comment by tonycoco** [comment]
This looks great, @koriroys, but I'm going to use a generic name for the script, something like `before_hook.sh` and also have an `after_hook.sh`

**Comment by koriroys** [comment]
:+1:

**Comment by tonycoco** [comment]
Want to give that a crack, @koriroys? I'll merge it then.

**Comment by koriroys** [comment]
@tonycoco seems to work with my app. Thoughts?

File: README.md [hunk 1]
```diff
@@ -53,6 +53,43 @@ For most Ember applications that make any kind of authenticated requests (sendin
...
+#### Before and After Hooks
+
+You can run your own scripts by creating `after_hook.sh` or `before_hook.sh` files (or both) in your app's `hooks` directory:
+
+    mkdir hooks
+    cd hooks
+    touch after_hook.sh
+    touch before_hook.sh
+
+See the section on compass for an example.
```
Comment (tonycoco) [inline README.md]: Capitalize Compass

File: README.md [hunk 2]
```diff
@@ -53,6 +53,43 @@ For most Ember applications that make any kind of authenticated requests (sendin
...
+
+#### Compass
+
+If you want to compile your compass assets as part of the build process, first create `after_hook.sh` in the `hooks` directory (see Before and After Hooks section), then add this code to it:
```
Comment (tonycoco) [inline README.md]: make it...

`(See: Before and After Hooks section)`

File: bin/compile [hunk 3]
```diff
@@ -178,5 +183,10 @@ if test -d $build_dir/bower_components; then
   cp -r $build_dir/bower_components $cache_dir/
 fi
 
+if test -f $build_dir/hooks/after_hook.sh; then
+  status "After Hook detected. Here comes the crocodile..."
```
Comment (tonycoco) [inline bin/compile]: Maybe something like, "After hook detected. Running..." same thing for the before hook.

**Comment by tonycoco** [comment]
Nevermind, @koriroys... I'll merge it and fix it later :)

**Comment by koriroys** [comment]
After Hook, crocodile reference: http://youtu.be/GYob5-pBxlM?t=55s

**Pull request merged**